meeting-notes/50-07May2020.md

```diff
@@ -0,0 +1,97 @@
+# dat protocol Community communication
+
+07 May 2020
+
+## People
+
+* Diego (@dpaez)
+* Mauve
+* @lidel
+* dietrich
+* jan (@falafeljan)
+* nina
+* Paul
+* [bret](https://bret.io)
+* [Franz](https://github.com/Frando)
+* substack
+* Davie
+
+## Meeting Notes
+
+### Intro
+
+- Diego: Works at GEUT. Working on integrating hyperdrive-daemon with p2p-commons SDK.
+- Paul: Works on Beaker Browser. Got the Beaker Beta to be released in about a week. Wrapping up the final pieces.
+- Sam: Working at Cliqz, browser company that's shutting down. There will still be a core left over so could maybe continue on the Dat stuff, the dat protocol in firefox based browsers. Latest is getting Dat v2 working there. Got the new hyperswarm and hyperdrive version working there. Will be compatible with Beaker when that's out.
+- Nina: Working at playproject.io Working on DatDot, a p2p hosting solution, everyone that has empty space will be able to offer it, in exchange they'll get hosting when they're offline. Currently working on MVP, working with Mauve. Exciting, getting there and it's a lot of work.
+- mauve: working on datdot with nina, also working on natakanu, for a local indigenous group. Also working with ipfs on the companion, adding mdns discovery to the brave browser companion. Need to touch some c++. Dat uses the same MDNS stuff so it could be useful if we ever get an extension
```

Comment:
```suggestion
- mauve: working on datdot with nina. also working on natakanu, file transfer app for indigenous communities in canada sponsored by Wapikoni Mobile. Also working with ipfs on an ipfs-companion, fixing mdns discovery in brave browser. Need to touch some c++ for Chromium patch. Dat uses the same MDNS stuff so it could be useful if we ever make an extension.
```